food-truck-frontend/src/components/notifications/NotificationWatcherComponent.tsx

```diff
@@ -32,7 +30,14 @@ function NotificationWatcherComponent(props: NotificationListProps) {
     return () => {};
   });
   return (
-    <></>
+    <Dialog open={notify && !notified}>
+      <DialogTitle>You have unread notifications!</DialogTitle>
+      <DialogContent>
+        <Grid container direction="column" alignItems="center">
+          <Button variant="contained" onClick={() => setNotified(true)}>OK</Button>
```

Comment:
If we're going to have a special component, maybe we should have a button to go to notifications? Just a thought.